docs/omnipaxos/reconfiguration.md

```diff
@@ -21,7 +21,7 @@ let decided_entries: Option<Vec<LogEntry<KeyValue>>> = omnipaxos.  read_decided_
 if let Some(de) = decided_entries {
     for d in de {
         match d {
-            LogEntry::StopSign(stopsign) => {
+            LogEntry::StopSign(stopsign, _stopsign_is_decided) => {
```

Comment:
This is correct since we are doing read_decided_suffix before. But for the sake of being clear in the docs, we should emphasize that a new instance should only be started when the stopsign is really decided. So here we should do a check that this flag is true first.